Makefile

```diff
@@ -84,6 +104,17 @@ docker-push: ## Push docker image with the manager.
 photon:
 	docker build -t github.com/vmware-tanzu/nsx-operator -f build/image/photon/Dockerfile .
 
+.PHONY: clean
+clean:
+	@rm -rf $(BINDIR)
+	@rm -rf $(GOLANGCI_LINT_BINDIR)
+	@rm -rf $(CURDIR)/.coverage
+
+.PHONY: verify
+verify:
+	@echo "===> Verifying spellings <==="
+	GO=$(GO) $(CURDIR)/hack/verify-spelling.sh
```

Comment:
Where's the `verify-spelling.sh` from? It looks that we didn't put it in the repo?

Reply:
will add it。